assembl/static/js/app/models/attachments.js

```diff
@@ -214,8 +215,12 @@ var AttachmentModel = Base.Model.extend({
         that = this;
     return d.destroy({
       success: function(model, response){
-        console.log('in document destroy success callback');
         return Base.Model.prototype.destroy.call(that);
+      },
+      error: function(model, response){
+        console.warn("Could NOT destroy document model id " + d.id
+                     + "\nContinuing to destroy attachment id " + this.id);
+        return Base.Model.prototype.destroy.call(that); 
```

Comment:
Hum...that's interesting. This error callback still tries to destroy the attachment model, even if the document model is not destroyed. So, from user perspective, the attachment would not be visible anymore. `console.warn` will be caught by Sentry, and that is something that we as developers would see.

But, I see your point. If the server is down, and the document model is not destroyed, AND attachment model is also not, then a growl or something should be shown to the user. Maybe now a good time to figure out a way to throw events at an aggregator?